dotCMS/src/main/java/com/dotmarketing/portlets/calendar/business/EventAPIImpl.java

```diff
@@ -141,10 +126,10 @@ public Event find(String id, boolean live, User user, boolean respectFrontendRol
 	}
 
 	public Event findbyInode(String inode, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
-		Event ev = eventFactory.findbyInode(inode, user, respectFrontendRoles);
+		Event ev = eventFactory.findbyInode(inode, user, respectFrontendRoles); // todo: this method should be here, since it is an API call and convert.

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)